skbio/io/format/embl.py

```diff
@@ -1352,17 +1320,27 @@ def _serialize_sequence(obj, indent=5):
     obj : DNA, RNA, Sequence Obj
     '''
 
-    # magic numbers
-    n = 1
-    line_size = 60
+    # a flag to determine if I wrote header or not
+    flag_header = False
+
+    # magic numbers: there will be 60 letters (AA, bp) on each line
+    chunk_size = 60
+
+    # letters (AA, bp) will be grouped by 10: each group is divided by
+    # one space from each other
     frag_size = 10
 
-    # count bases in sequence
+    # this is the final line length
+    line_size = 80
```

Comment:
is the line size fixed? what if the sequence is really long and `60 + number of spaces + number of digits (index on the right)` larger than 80?